homework2.js

```diff
@@ -0,0 +1,100 @@
+/*
+1. Задача про обчислення різниці часу
+ */
+
+function durationBetweenDates( date1, date2, units='seconds' ) {
+	let smallerDate = Math.min( Date.parse( date1 ), Date.parse( date2 ));
+	let biggerDate = Math.max( Date.parse( date1 ), Date.parse( date2 ));
+	//let [ smallerDate, biggerDate ] = [ Date.parse( date1 ), Date.parse( date2 ) ].sort( ( a, b ) => a - b );
+	if( !smallerDate || !biggerDate ) {
+		console.log( 'Wrong date format' );
+		return;
+	}
+	const result = biggerDate - smallerDate;
+	switch ( units ) {
+		// 'days', 'hours', 'minutes', seconds)
+		case 'days':
+			console.log( `${ Math.floor(result/1000/60/60/24 ) } days` );
+			break;
+		case 'hours':
+			console.log( `${result/1000/60/60} hours` );
```

Comment:
Все що дублюється, тобто /1000 також можна і треба винести в 13 рядок

І ще, функція має повертати значення, а не виводити його в консоль, тобто тут має бути:
```suggestion
			return `${result/1000/60/60} hours`;
```

Reply:
Done